xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-ui/src/XWiki/Lightbox/WebHome.xml

```diff
@@ -0,0 +1,838 @@
+        <name>parameters</name>
+        <number>7</number>
+        <prettyName>Extension Parameters</prettyName>
+        <rows>10</rows>
+        <size>40</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </parameters>
+      <scope>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>scope</name>
+        <number>8</number>
+        <prettyName>Extension Scope</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators>|, </separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>wiki=Current Wiki|user=Current User|global=Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </scope>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <content>{{velocity}}
+#set ($lightboxConfigDoc = $xwiki.getDocument('XWiki.Lightbox.LightboxConfiguration'))
+#set ($lightboxConfigObj = $lightboxConfigDoc.getObject('XWiki.Lightbox.LightboxConfigurationClass'))
+#if ($lightboxConfigObj.getValue('isLightboxEnabled').equals(1))
+  #set ($discard = $xwiki.linkx.use($services.webjars.url('org.xwiki.contrib:webjar-blueimp-gallery', 'css/blueimp-gallery.min.css'), {'type': 'text/css', 'rel': 'stylesheet'}))
+  #set ($discard = $xwiki.jsx.use('XWiki.Lightbox.WebHome'))
+  #set ($discard = $xwiki.ssx.use('XWiki.Lightbox.WebHome'))
+  {{html clean="false"}}
+  &lt;!-- The Gallery lightbox dialog. -->
+  &lt;div
+    id="blueimp-gallery"
+    class="blueimp-gallery"
+    aria-label="image gallery"
+    aria-modal="true"
+    role="dialog"
+  >
+    &lt;div class="slides" aria-live="polite">&lt;/div>
+    &lt;h3 class="title">&lt;/h3>
+    &lt;a
+      class="prev" aria-controls="blueimp-gallery"
+      aria-label="previous slide" aria-keyshortcuts="ArrowLeft"
+    >&lt;/a>
+    &lt;a
+      class="next" aria-controls="blueimp-gallery"
+      aria-label="next slide" aria-keyshortcuts="ArrowRight"
+    >&lt;/a>
+    &lt;div class="controls btn-group">
+      &lt;a
+        class="btn fullscreen" id="lightboxFullscreen" aria-controls="blueimp-gallery"
+        aria-label="fullscreen" aria-keyshortcuts="Fullscreen"
+      >$services.icon.renderHTML('arrows')&lt;/a>
+      &lt;a
+        class="btn download" id="lightboxDownload" aria-controls="blueimp-gallery"
+        aria-label="download" aria-keyshortcuts="Download"
+      >$services.icon.renderHTML('download')&lt;/a>
+      &lt;a
+        class="btn autoPlay" aria-controls="blueimp-gallery" aria-label="autoPlay slideshow"
```

Comment:
Shouldn't these labels be translated? Further, wouldn't it make sense to set this also as `title`-attribute so users that aren't using assistive technologies can also discover the functionality of the button?